File: test/aidbox_sdk/snapshots/dotnethl7-fhir-r4-core/Practitioner.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.R4.Core;

public class Practitioner : DomainResource, IResource
{
    public Base.Address[]? Address { get; set; }
    public Base.HumanName[]? Name { get; set; }
    public string? BirthDate { get; set; }
    public Base.Attachment[]? Photo { get; set; }
    public bool? Active { get; set; }
    public Base.CodeableConcept[]? Communication { get; set; }
    public Base.Identifier[]? Identifier { get; set; }
    public Practitioner_Qualification[]? Qualification { get; set; }
    public Base.ContactPoint[]? Telecom { get; set; }
    public string? Gender { get; set; }

    public class Practitioner_Qualification : BackboneElement
    {
        public required Base.CodeableConcept Code { get; set; }
        public Base.ResourceReference? Issuer { get; set; }
        public Base.Period? Period { get; set; }
        public Base.Identifier[]? Identifier { get; set; }
    }
}

File: test/aidbox_sdk/snapshots/python/hl7_fhir_r4_core/Practitioner.py
from typing import Optional, List
from dataclasses import dataclass, field
from base import Address
from base import Attachment
from base import Period
from base import CodeableConcept
from base import ContactPoint
from base import HumanName
from base import DomainResource
from base import Reference
from base import Identifier
from base import BackboneElement

@dataclass(kw_only=True)
class PractitionerQualification(BackboneElement):
    code: CodeableConcept
    issuer: Optional[Reference] = None
    period: Optional[Period] = None
    identifier: Optional[List[Identifier]] = field(default_factory=list)

@dataclass(kw_only=True)
class Practitioner(DomainResource):
    address: Optional[List[Address]] = field(default_factory=list)
    name: Optional[List[HumanName]] = field(default_factory=list)
    birth_date: Optional[str] = None
    photo: Optional[List[Attachment]] = field(default_factory=list)
    active: Optional[bool] = None
    communication: Optional[List[CodeableConcept]] = field(default_factory=list)
    identifier: Optional[List[Identifier]] = field(default_factory=list)
    qualification: Optional[List[PractitionerQualification]] = field(default_factory=list)
    telecom: Optional[List[ContactPoint]] = field(default_factory=list)
    gender: Optional[str] = None